camunda-formio-tasklist-vue/src/components/FormListModal.vue

```diff
@@ -126,10 +135,11 @@ export default class FormListModal extends Mixins(BaseMixin) {
   private formcurrentPage: number = 1;
   private formValueId: string | null = null;
   private formId: string | null = null;
-  private submissionId: string | null = null;
   private formioUrl: string | null = null;
   private formTitle: string = "";
   private formsflowAIApiUrl: any;
+  private sortAscending: boolean = false;
+  private submissionId: string | null = null;
```

Comment:
if there is a chance for the variable to be null or undefined, then you can do `private variableName?: string = value/null/undefined;`

Reply:
Thanks for suggesting the change. Incorporating it throughout the codebase as well.

Isn't this rule application even for type hints in function definition?

Reply:
![image](https://user-images.githubusercontent.com/70306694/129292913-785eea2d-d0d4-4872-ae35-9f480e97a2db.png)

getting this error when converting to this format